enterprise/cli/licenses.go

```diff
@@ -187,3 +192,20 @@ func (r *RootCmd) licenseDelete() *clibase.Cmd {
 	}
 	return cmd
 }
+
+func convertLicenseExpireTime(license codersdk.License) codersdk.License {
+	if license.Claims["license_expires"] != nil {
+		value, ok := license.Claims["license_expires"].(json.Number)
+		if !ok {
+			return license
+		}
+		int64Value, err := value.Int64()
+		if err != nil {
+			return license
+		}
+		t := time.Unix(int64Value, 0)
+		rfc3339Format := t.Format(time.RFC3339)
+		license.Claims["license_expires"] = rfc3339Format
```

Comment:
It works accidentally as maps in Go are passed by reference. Could you please create a copy of the Claims object first, then modify, and reassign it?